kotlin/injivcrenderer/src/commonMain/kotlin/io/mosip/injivcrenderer/qrCode/QrCodeGenerator.kt

```
@@ -2,24 +2,27 @@ package io.mosip.injivcrenderer.qrCode
     private val className = QrCodeGenerator::class.simpleName
 
-    fun generateQRCodeImage( vcJson: String): String {
+    fun generateFromVcJson(vcJson: String): String {
         try {
-            val pixelPass = PixelPass()
-            val qrData: String = pixelPass.generateQRData(vcJson)
-
-            return convertQrDataIntoBase64(qrData)
-
+            return generateQrCodeFromVcJson(vcJson)
         } catch (e: Exception){
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)